Solutions/STM32F4DISCOVERY/STM32F4DISCOVERY.settings

```diff
@@ -16,6 +16,7 @@
     <IsSolutionWizardVisible>True</IsSolutionWizardVisible>
     <ENDIANNESS>le</ENDIANNESS>
     <NO_BOOTLOADER_COMPRESSION>true</NO_BOOTLOADER_COMPRESSION>
+    <PLATFORM_EMULATED_FLOATINGPOINT Condition="'$(COMPILER_TOOL)'=='GCC'">true</PLATFORM_EMULATED_FLOATINGPOINT>

```

Comment:
Actually this because of the crypto libs. GCC link complains about a mismatch in the VFP flags between the crypto libs and the netmf. This has been discussed a number of times on other threads.